rest-api-spec/src/main/resources/rest-api-spec/api/clear_scroll.json

```diff
@@ -21,8 +21,7 @@
           "parts":{
             "scroll_id":{
               "type":"list",
-              "description":"A comma-separated list of scroll IDs to clear",
-              "deprecated":true
```

Comment:
Is removing `deprecated` here intended?

Reply:
Yes it is. It's covered by the `"deprecated"` at the parent path level:

https://github.com/elastic/elasticsearch/blob/4d086f2ccd3c510cc2d6b4c192519c42c70c9baa/rest-api-spec/src/main/resources/rest-api-spec/api/clear_scroll.json#L27-L30

There are a few cases where I've removed these

Reply:
We had a call to discuss the removal of the `"deprecated": bool` properties on parts and params. I'm going to reinstate the properties for now and add `"deprecated"` to the json schema, with a view to discuss removing these in the future.